server/server_test.go

```diff
@@ -157,17 +160,24 @@ func (suite *ServerTestSuite) TestIssueRedeemV2() {
 	err = json.Unmarshal(body, &issuerResp)
 	suite.Require().NoError(err, "Redemption response body unmarshal must succeed")
 	suite.Assert().NotEqual(issuerResp.Cohort, 1-issuerCohort, "Redemption of a token should return the same cohort with which it was signed")
-
-	_, _ = suite.srv.db.Query(`UPDATE issuers SET expires_at=$1 WHERE id=$2`, time.Now().AddDate(0, 0, -1), issuer.ID)
-	issuers, _ := suite.srv.fetchIssuers(issuerType)
-	suite.Assert().Equal(len(*issuers), 2, "There should be two issuers of same type")
+	_, err = suite.srv.db.Query(`UPDATE v3_issuers SET expires_at=$1 WHERE issuer_id=$2`, time.Now().AddDate(0, 0, -1), issuer.ID)
+	suite.Require().NoError(err, "failed to expire issuer")
+	// keys are what rotate now, not the issuer itself
+	// issuers, _ := suite.srv.fetchIssuers(issuerType)
```

Comment:
Do need to keep this commented code

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/c212d77eac67c1cc0369b89c95e4dea5812fd3fe